core/domain/rating_services.py

```diff
@@ -95,17 +107,34 @@ def get_user_specific_rating_for_exploration(user_id, exploration_id):
 
 
 def get_when_exploration_rated(user_id, exploration_id):
-    """Returns the date-time the exploration was lasted rated by this user, or
-    None if no such rating has been awarded.
+    """Fetches the date-time the exploration was last rated by this user, or
```

Comment:
date-time --> datetime (just for consistency with how we refer to this in the codebase in general)